draft-ietf-dnsop-dns-catalog-zones.md

```diff
@@ -385,15 +394,15 @@ refresh timeout, so care must be taken to make the member zones available
 before any update to the list of member zones is visible in the catalog zone.
 
 When zones are deleted from a catalog zone, a primary MAY delete the member
-zone immediately after notifying secondaries.  It is up to the secondary
+zone immediately.  It is up to the secondary
 nameserver to handle this condition correctly.
 
 {#zonereset}
 When the `<m-unique-N>` label of a member zone changes, all its associated state MUST be reset, including the zone itself.
 This can be relevant for example when zone ownership is changed.
 In that case one does not want the new owner to inherit the metadata.
```

Comment:
Like data associated with the previous owner (need to explain this to peter Thomassen)

Reply:
Or the draft needs to explain it, because I am also confused about it still :)